lib/hanami/model/configuration.rb

```diff
@@ -225,50 +43,24 @@ def root
         Hanami.respond_to?(:root) ? Hanami.root : Pathname.pwd
```

Comment:
This should stay `pwd` because in configuration we use to do:

```ruby
Hanami::Model.configure do
  # ...
  migrations 'db/migrations'
  schema     'db/schema.sql'
end
```

As you can see, that are relative paths, that joined with `pwd` return the correct directory/file.